ee/vellum_ee/workflows/display/utils/vellum.py

```diff
@@ -124,11 +124,6 @@ def create_node_input_value_pointer_rule(
         child_descriptor = get_child_descriptor(value, display_context)
         return create_node_input_value_pointer_rule(child_descriptor, display_context)
     if isinstance(value, WorkflowInputReference):
-        if value not in display_context.global_workflow_input_displays:
-            raise ValueError(
-                f"Serialization failed: type object '{value.inputs_class.__qualname__}' "
-                f"has no attribute '{value.name}'"
-            )
```

Comment:
oh we still want something like this. Extend the test below to have both a templating node and a base node reference the invalid input ref, which will force us to raise this